sql-db/sql-app/src/test/java/io/quarkus/ts/sqldb/sqlapp/DB2DatabaseIT.java

```diff
@@ -11,6 +11,7 @@
 
 @QuarkusScenario
 @Tag("fips-incompatible") // Reported in https://github.com/IBM/Db2/issues/43
+// TODO: once DB2 is FIPS-compatible, move quarkus-jdbc-db2 from own Maven profile so that the extension is available in FIPS
```

Comment:
Suggestion: move this todo to the pom and link the github issue there

Reply:
I'll move this TODO to the POM file and link GH issue, np